### ISSUE TITLE
Share image for Twitter/X

### ISSUE THREAD

**Issue by joshmedeski** [ghostty-org/website#132]
Hey, just noticed sharing Ghostty doesn't render a social image.

You want me to design one and add the appropriate HTML tags to the docs site to support most social media sites?

 <img width="610" alt="Image" src="https://github.com/user-attachments/assets/bf152f3b-ed35-410b-96fb-ccd059610901" />

**Comment by mitchellh** [comment]
Hm I thought we did this, @BrandonRomano cause I thought you did this. Just want to check before someone else does. 😄

**Comment by semanticentity** [comment]
Add twitter:image, not just og:image

![Image](https://github.com/user-attachments/assets/a9d1eaaa-c241-4df9-854f-49d173a51748)

**Comment by BrandonRomano** [comment]
Yeah we added a social image; I can add `twitter:image` as well.

**Comment by BrandonRomano** [comment]
Okay; so it looks like twitter's cache is holding onto no image.  I'm going to keep this open for a bit.  We can re-close the issue after we've validated the cache is cleared.

It looks like twitter got rid of their [Validator](https://cards-dev.x.com/validator), which used to be able to clear cache for you.

There's a [third-party tool here](https://threadcreator.com/tools/twitter-card-validator) which suggests I've configured it correctly.

![Image](https://github.com/user-attachments/assets/a6b0e91b-3966-4822-b738-10196c3668b0)
